lib/lib.pro

```diff
@@ -0,0 +1,42 @@
+TEMPLATE = lib
+TARGET = amberwebauthorization
+TARGET = $$qtLibraryTarget($$TARGET)
+TARGETPATH = $$[QT_INSTALL_LIBS]
+
+include(../defaults.pri)
+
+CONFIG += qt create_pc create_prl no_install_prl link_pkgconfig
+QT += qml network
```

Comment:
Is `qml` requirement necessary for library which supposed to provide C++ API only?

Reply:
no, not required.  now removed, and also from pkgconfig below.